build.gradle

```diff
@@ -20,6 +20,8 @@ ext.scenariooAggregatedDataFormatVersion = '2.1.0'
  */
 ext.scenariooReleaseBranch = 'release-3.0'
 
+ext.scenariooVersion = '3.0.1-SNAPSHOT'
```

Comment:
I don't see a good reason to use the git commit id. We agreed on using semantic version. Our artifacts should be named accordingly. It only makes our life harder (e.g. we always have to produce a scenarioo-latest to be able to deploy our CI).